libcontainer/cgroups/systemd/v2.go

```diff
@@ -301,9 +300,11 @@ func (m *unifiedManager) Destroy() error {
 		return err
 	}
 
-	// XXX this is probably not needed, systemd should handle it
-	err := os.Remove(m.path)
-	if err != nil && !os.IsNotExist(err) {
+	// systemd 239 do not remove sub-cgroups.
+	logrus.Debugf("try to destroy cgroup: %s", m.path)
```

Comment:
I saw there was a comment about this earlier; mostly wondering how useful this debug log is, as it would be logging the "happy" path as well (successfully handled this); the "failure" path is already returned as an error (which may be more interesting in case things don't work as expected).

Is there a strong motivation to log this case?

Reply:
Thanks for your suggestion. When I met the error message, I want to make sure if this code is the root cause. I have deleted the debug log.

Reply:
Thanks! If the error message itself is not providing enough information, we could add a (debug) log _inside_ the `if err != nil`. I _suspect_ the error will contain information about the path that failed to be removed though (so perhaps not needed to add additional logs for that)